### PR TITLE
Updated basics of IN-CORE to include other services

### PR DESCRIPTION
**Pull request by navarroc**
Added more description to the basics of IN-CORE section about the other two services and mentioned utilities for post processing. I tried to keep it simple so it's not too wordy. Let me know what you think and if we should add anything else there.

### CODE REVIEW

**Review by ywkim312 — APPROVED** [review]
I am not very sure if I am the one who is suitable to review this PR since this is mostly English thing :) Anyway, it looked good to me so I approve. Just one thing, we don't need to mention space service in here, right?

File: in-core_joplin_case_study.ipynb [hunk 1]
```diff
@@ -66,9 +66,21 @@
...
+    "  - Request deletion of a hazard\n",
+    "- **DFR3Service:** similar to the **HazardService**, once the `IncoreClient` has been initiated, the `DFR3Service` module uses the \n",
+    "  information from that client to administer existing information about known or new fragility, restoration, repair and recovery curves.     `DFR3Service` can:\n",
+    "  - Retrieve existing curves, mappings and metadata \n",
+    "  - Search fragility curves and mappings per specific information\n",
+    "  - Request creation of new curves and mappings\n",
+    "  - Request deletion of a curves and mappings\n",
+    "- **DataService:** similar to the **HazardService**, once the `IncoreClient` has been initiated, the `DataService` module uses the \n",
+    "  information from that client to administer existing information about known or new datasets. `DataService` can:\n",
+    "  - Retrieve existing metadata for various datasets\n",
```
Comment (longshuicy) [inline in-core_joplin_case_study.ipynb]: Data Service can also download the data. Would it make sense to add one more bullet point?

**Review by longshuicy — APPROVED** [review]

File: in-core_joplin_case_study.ipynb (hunk 1 above)
Comment (navarroc) [reply in-core_joplin_case_study.ipynb]: Done, I added it to the first bullet

**Comment by navarroc** [comment]
> I am not very sure if I am the one who is suitable to review this PR since this is mostly English thing :) Anyway, it looked good to me so I approve. Just one thing, we don't need to mention space service in here, right?

I wasn't going to since it's more of a secondary service that is utilized by the other services. I can if others think we should add that as well.
